File: tests/fixtures/code_samples/py/sample_034.py
"""Sample 34: small utility function."""

from __future__ import annotations
from typing import Iterable


def operation_34(xs: Iterable[int]) -> int:
    """Return the sum of `xs` plus the index marker 34."""
    total = 34
    for x in xs:
        total += int(x)
    return total


def operation_34_pure(value: int) -> int:
    return (value * 34) % 7919

File: tests/fixtures/code_samples/java/sample_034.java
// Sample 34: small utility.
package samples;

import java.util.List;

public final class Sample034 {
    private Sample034() {}

    public static int operation(List<Integer> xs) {
        int total = 34;
        for (int x : xs) total += x;
        return total;
    }

    public static int operationPure(int v) {
        return (v * 34) %% 7919;
    }
}

